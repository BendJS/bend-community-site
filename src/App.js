import React, { Fragment } from 'react';
import { Router } from '@reach/router';

import ContactUs from './components/contact-us';
import Home from './components/home';
import Header from './components/header';
import './App.css';

function App() {
  return (
    <Fragment>
      <Header />
      <Router>
        <Home path='/' />
        <ContactUs path='contact-us' />
      </Router>
    </Fragment>
  );
}

export default App;
