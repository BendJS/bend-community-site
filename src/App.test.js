import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect'
import App from './App';

describe('App', () => {
  it('renders without crashing', () => {
    const { getByTestId } = render(<App />);

    expect(getByTestId('app')).toMatchSnapshot();
  });
});
