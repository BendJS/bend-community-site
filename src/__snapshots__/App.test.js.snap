// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  class="makeStyles-root-1"
  data-testid="app"
>
  <header
    class="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <h6
          class="MuiTypography-root makeStyles-title-2 MuiTypography-h6"
        >
          Bend Community Site
        </h6>
        <a
          aria-current="page"
          aria-disabled="false"
          class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-3 MuiButton-colorInherit"
          href="/"
          role="button"
          tabindex="0"
        >
          <span
            class="MuiButton-label"
          >
            Home
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-3 MuiButton-colorInherit"
          href="/contact-us"
          role="button"
          tabindex="0"
        >
          <span
            class="MuiButton-label"
          >
            Contact Us
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
      </div>
    </div>
  </header>
</div>
`;
